.github/workflows/flatpak-update.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Flatpak YAML to use latest version of ASIMOV CLI
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * 4"
```

Comment:
Always include a comment ("At 16:00 every Thursday") with a Crontab.guru link (https://crontab.guru/#0_16_*_*_4) for any crontab spec. Also, this should run more often than once a week. Make it nightly

Reply:
@artob it is done in - https://github.com/asimov-platform/asimov-packaging/pull/6/commits/f32116ad765ef6328d08e3916194ad8617e0be99